exporter/elasticsearchexporter/README.md

```diff
@@ -145,6 +145,9 @@ This can be customised through the following settings:
   - `prefix_separator`(default=`-`): Set a separator between logstash_prefix and date.
   - `date_format`(default=`%Y.%m.%d`): Time format (based on strftime) to generate the second part of the Index name.
 
+- `logs_dynamic_id` (optional): Dynamically determines the document ID to be used in Elasticsearch based on resource, scope, or log record attributes.
+  - `enabled`(default=false): Enable/Disable dynamic ID for log records. If `elasticsearch.document_id` exists in attributes (precedence: log record attribute > scope attribute > resource attribute), it will be used as the document ID. Otherwise, the document ID will be generated by Elasticsearch.
```

Comment:
I think it should only ever be taken from log record attributes. If it were specified in the scope or resource attributes, then multiple log records would get the same doc ID and would be considered duplicates.

Reply:
Definitely makes sense now that I think of it, I'll update the PR to only consider the log record attrs.